bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+
+import h5py
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from collections import OrderedDict
```

Comment:
Is `OrderedDict` used at any point? I don't see it being called, it might be leftover from the old version.

Reply:
Correct, will remove